src/chainparams.cpp

```diff
@@ -274,12 +279,12 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 1199145601; // January 1, 2008
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = 1230767999; // December 31, 2008
         consensus.nStakeMinAge = 2;	// minimum for coin age: 2 seconds
-        consensus.nTargetSpacing = 30; // Blocktime: 30 secs
+        consensus.nTargetSpacing = 1; // Blocktime: 1 secs
```

Comment:
```suggestion
        consensus.nTargetSpacing = 30; // Blocktime: 30 secs
```
We need to switch this back don't we?